### ISSUE TITLE
icb_parpack_cpp test is failing on ppc64el

### ISSUE THREAD

**Issue by sylvestre** [opencollab/arpack-ng#128]
https://buildd.debian.org/status/fetch.php?pkg=arpack&arch=ppc64el&ver=3.6.1-1&stamp=1529756139&raw=0

```
rank 0 - 1000.15 1000.15
Correct eigenvalues not computed
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
FAIL icb_parpack_cpp (exit status: 1)
```

**Comment by sylvestre** [comment]
@fghoussen Does it ring a bell?
we could just disable mpi on this arch...

**Comment by fghoussen** [comment]
Not really: soundless bell ! In build dependencies, I see libopenmpi**3**: could you try with openmpi**2** ?

**Comment by fghoussen** [comment]
If switching from openmpi2 to openmpi3 is OK on ppc64el (?), try it also on armel

**Comment by fghoussen** [comment]
If libopenmpi3 is not to blame (?), then C++11 could be to blame....

 I've just seen in the (armel) log (at the end) :
```
libtool: link: mpic++ -std=gnu++11 ...
E: Build killed with signal TERM after 150 minutes of inactivity

```
The build seems to hang. Looks like the compiler goes crazy because he doesn't know what to do with the (hard-coded) gnu++11 ?!... Really not sure (what is the compiler behind ?). I have no good answer here. I PR a few lines: kill them if they don't go the way you like

**Comment by 10110111** [comment]
Looks more like a test hanging than the compiler disliking `gnu++11`. The compiler seems to have finished by the time the timeout message is printed.

**Comment by fghoussen** [comment]
Yeah maybe. Can you have more verbose logs ? Or compile with ` make VERBOSE=1` ?

**Comment by 10110111** [comment]
I guess not. But from the log it seems that the first test in the MPI directory hangs. I suppose only adding some prints to the test would help debugging.

**Comment by dbeurle** [comment]
I'm not entirely sure about this, but in the CMake it links against the CXX and the C libraries for MPI.  Is this likely a problem?

Other possibility is a dead lock :/

**Comment by fghoussen** [comment]
Just realized hang is after `issue46` test... Which is **not** an icb one ?! No idea what's going on

**Comment by fghoussen** [comment]
@sylvestre: I guess the problem on armel/ppc64 _may_ be fixed by 9742da75dc2f7 !....

**Comment by fghoussen** [comment]
@sylvestre : if you're about to publish a new release, ckeckout if this issue is (should be !) fixed

**Comment by fghoussen** [comment]
May be fixed by f36eb6c75dc686c8cebdaca0106422d77b04152d

**Comment by fghoussen** [comment]
> May be fixed by [f36eb6c](https://github.com/opencollab/arpack-ng/commit/f36eb6c75dc686c8cebdaca0106422d77b04152d)

@sylvestre: if #397 is released would be good to check if this problem is solved

**Comment by sylvestre** [comment]
yeah, i will upload it once 3.9.1 is tagged :)
